README.md

```diff
@@ -105,6 +105,17 @@ except_tables = [
 MysqlRewinder.setup(db_configs, except_tables: except_tables)
 ```
 
+### Logging
+
+If you want to enable logging, do the following:
+
+* Generate db_configs from `ActiveRecord::Base.configurations`
+* Pass `logger: Logger` to `MysqlRewinder.setup`.
+
+```ruby
+MysqlRewinder.setup(db_configs, logger: Logger.new(STDOUT))
+```
```

Comment:
I think we can simplify like:

```suggestion
If you want to enable logging, specify `logger` for `MysqlRewinder.setup`

```ruby
MysqlRewinder.setup(db_configs, logger: Logger.new(STDOUT))
```
```